Sources/Danger/BitBucketCloud.swift

```diff
@@ -228,6 +228,22 @@ extension BitBucketCloud {
 
         /// The user that created the comment
         public let user: User
+        
+        public init(from decoder: Decoder) throws {
+            let container = try decoder.container(keyedBy: CodingKeys.self)
+            
+            self.content = try container.decode(Content.self, forKey: .content)
+            self.createdOn = try container.decode(Date.self, forKey: .createdOn)
+            self.deleted = try container.decode(Bool.self, forKey: .deleted)
+            self.id = try container.decode(Int.self, forKey: .id)
+            // protect from "inline": {}
+            self.inline = (try? container.decodeIfPresent(Inline.self, forKey: .inline)) ?? nil
```

Comment:
You can just make `path` optional too, and that prevents from `"inline": {}` as well, but also from all the combinations that you can have (only path, only from etc).
If, on the other hand, it either have only zero or all the variables, I would then make them all required, and keep this line, what are your thoughts about this?

Reply:
Clearly from the json I printed out.. To was set to null.  so I don't think you can make them all required.

Decoding JSON is like a losing battle if you don't control it ;)

Reply:
I would be ok I guess with having everything in `inline` be optional

Reply:
Yeah looks a good option :)

Reply:
ok.. so we rather just make all of the fields in inline optional vs the init right?

Reply:
👍🏻